src/components/FilmsList.vue

```diff
@@ -58,18 +58,19 @@ export default {
     },
     async fetchFilms() {
       this.loading = true;
```

Comment:
Same comment here, use the "isThingTrue" naming convention when naming component's own boolean properties.